templates/shortcode-receipt.php

```diff
@@ -30,7 +30,7 @@
 
 // Update donor name, if title prefix is set.
 if ( ! empty( $user['title'] ) ) {
-	$full_name    = "{$user['title']}. {$user['first_name']} {$user['last_name']}";
+	$full_name    = "{$user['title']} {$user['first_name']} {$user['last_name']}";
```

Comment:
@mehul0810 last name and title prefix are not required, so they can be empty, use time when you create name string.